src/main/java/lukemccon/airdrop/controllers/PackagesController.java

```diff
@@ -18,8 +20,7 @@ public static boolean onCommand(CommandSender sender, String[] args) {
 		// Lists available packages
 		// /airdrop packages
 
-		String list = PackageManager.list();
-		ChatHandler.sendMessage(sender, list);
+		Airdrop.PACKAGES_GUI.openInventory((Player) sender);
```

Comment:
It doesn't look like we check that the sender is a player here before trying to cast them to a Player. If the server console sent this it would probably error here or error when it tries to open an inventory.